po/it.po

```diff
@@ -458,42 +450,39 @@ msgstr "Incolla inserendo pagine _pari"
 
 #: data/menu.ui.h:27
 msgid "Insert Blan_k Page"
-msgstr ""
+msgstr "Inserisci pagina vuota"
 
 #: data/menu.ui.h:28
 msgid "_Select"
-msgstr ""
+msgstr "_Seleziona"
 
 #: data/menu.ui.h:29
 msgid "Select _All"
-msgstr ""
+msgstr "_Seleziona_tutto"
 
 #: data/menu.ui.h:30
 msgid "_Deselect All"
-msgstr ""
+msgstr "_Deseleziona Tutto"
 
 #: data/menu.ui.h:31
-#, fuzzy
 msgid "Select _Odd Pages"
-msgstr "Taglia pagine selezionate"
+msgstr "Seleziona_Pagine dispari"
```

Comment:
The accelerator symbol (_) does not replace a space, we need a space in order to display this as separate words (same as in the english string).